prac2-pages/Simranpreet Kaur/main.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'login_page.dart'; 
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    
+    return ScreenUtilInit(
```

Comment:
Efficient thinking! :100: 
Trying to wrap the ScreenUtil over MaterialApp so that it automatically can be used in all the child widgets! :+1: